rclpy/rclpy/node.py

```diff
@@ -761,6 +763,33 @@ def _set_parameters_atomically(
 
         return result
 
+    def add_on_set_parameters_callback(
+        self,
+        callback: Callable[[List[Parameter]], SetParametersResult]
+    ) -> None:
+        """
+        Add a callback in front to the list of callback.
+
+        Calling this function will add a callback in self._parameter_callbacks list.
+
+        :param callback: The function that is called whenever parameters are set for the node.
+        """
+        self._parameters_callbacks.insert(0, callback)
+
+    def remove_on_set_parameters_callback(
+        self,
+        callback: Callable[[List[Parameter]], SetParametersResult]
+    ) -> None:
+        """
+        Remove a callback from list of callback.
```

Comment:
Spelling: ... list of callbackS.

Same below.

Reply:
@dirk-thomas okay sir I will update soon

Reply:
@dirk-thomas @allenh1 should I remove the `remove` api? 
but in c++ remove api is there.